### PR TITLE
EHR Workspace Creation and Management

### PR DESCRIPTION
**Pull request by musclepull**

### CODE REVIEW

File: package.json
Comment (MrDiggles2) [inline package.json:1]: remove if not using

File: ehr-integration/ehr-integration/src/app/interfaces/base.repository.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import axios, { AxiosInstance } from 'axios';
+import { IBaseRepository } from './IBaseRepository';
+import { Injectable, Logger } from '@nestjs/common';
+import { AccessTokenService } from '../app-modules/i-know-meds/access-token/access-token.service';
+import { ErrorType } from '../helpers/error.helper';
+
+@Injectable()
+export class BaseRepository<T, TId> implements IBaseRepository<T, TId> {
```
[the remark this answers is not in the document]
Comment (musclepull) [reply ehr-integration/ehr-integration/src/app/interfaces/base.repository.ts:7]: Yeah, i was thinking about that. Was going through some mental gymnastics over the weekend about the pathway moving forward.

File: backend/apps/backend/src/app/app-modules/user/queries/handlers/get-user-by-user-id.handler.ts
Comment (MrDiggles2) [inline backend/apps/backend/src/app/app-modules/user/queries/handlers/get-user-by-user-id.handler.ts:1]: @musclepull - can you check to see how these changes got reverted? I had switched this in an earlier pr when we moved from db to auth0 for users.

File: docker-compose.yml [hunk 2]
```diff
@@ -30,18 +31,52 @@ services:
       - DATABASE_USERNAME=${DATABASE_USERNAME}
       - DATABASE_PASSWORD=${DATABASE_PASSWORD}
       - DATABASE_NAME=${DATABASE_NAME}
-      - AUTH0_MANAGEMENT_DOMAIN=${AUTH0_MANAGEMENT_DOMAIN}
-      - AUTH0_MANAGEMENT_CLIENT_ID=${AUTH0_MANAGEMENT_CLIENT_ID}
-      - AUTH0_MANAGEMENT_CLIENT_SECRET=${AUTH0_MANAGEMENT_CLIENT_SECRET}
```
Comment (MrDiggles2) [inline docker-compose.yml:35]: can you revert this change as well
